src/bitcoin-cli.cpp

```diff
@@ -16,6 +16,7 @@
 #include <utilstrencodings.h>
 
 #include <memory>
+#include <tuple>
```

Comment:
Minor nit:

```
2018-09-13 21:33:38 clang-tidy(pr=13883): src/bitcoin-cli.cpp:18:1: warning: #includes are not sorted properly [llvm-include-order]
```

Reply:
Would you prefer to put tuple after stdio.h?

Reply:
I don't think we separate C and C++ headers elsewhere, but please sort them.